### PR TITLE
Release 22.8

### PR DESCRIPTION
**Pull request by ivankamkin**

### CODE REVIEW

File: doc/deb_packages.list [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+golang-1.15-go
+golang-1.15-src
+golang-doc
+golang-go
+golang-src:amd64
+golint
+java-common
+java-wrappers
+javascript-common
+kubectl
```
Comment (Denis-Averin) [inline doc/deb_packages.list:28]: No need to install kubernetes )

File: scripts/install-all-packages.bash [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   echo ****Install  "$package"****
+   pkg_ok=$(dpkg-query -W --showformat='${Status}\n' $package|grep "install ok installed")
+   echo Checking for $package: $pkg_ok
+    if [ "" = "$pkg_ok" ]; then
+        echo "No $package. Setting up $package."
+        sudo apt-get -y install $package
+    fi
+done
+dotnet tool install -g dotnet-format
+composer global require friendsofphp/php-cs-fixer
```
Comment (Denis-Averin) [inline scripts/install-all-packages.bash:31]: This line belongs to `make init` in php submodule

File: scripts/install-all-packages.bash [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+do
+   echo ****Install  "$package"****
+   pkg_ok=$(dpkg-query -W --showformat='${Status}\n' $package|grep "install ok installed")
+   echo Checking for $package: $pkg_ok
+    if [ "" = "$pkg_ok" ]; then
+        echo "No $package. Setting up $package."
+        sudo apt-get -y install $package
+    fi
+done
+dotnet tool install -g dotnet-format
```
Comment (Denis-Averin) [inline scripts/install-all-packages.bash:30]: This line belongs to `make init` in dotnet submodule

File: doc/deb_packages.list (hunk 1 above)
Comment (ivankamkin) [reply doc/deb_packages.list:28]: Ok. It was in your package list

**Review by Denis-Averin — APPROVED** [review]

**Pull request merged**